packages/daisy/src/components/toast/toast.tsx

```diff
@@ -0,0 +1,47 @@
+import { component$ } from '@builder.io/qwik';
+import { Toast as HeadlessToast } from '@qwik-ui/headless';
+
+interface ToastProps {
+  class?: string;
+  top?: boolean;
+  center?: boolean;
+  end?: boolean;
+  middle?: boolean;
+  bottom?: boolean;
+  start?: boolean;
+  label?: string;
+  alert?: string;
+}
+
+export const Toast = component$((props: ToastProps) => {
+  const { label, top, start, center, end, middle, bottom, alert, ...rest } =
+    props;
+
+  return (
+    <div class="form-control">
+      <label class="label cursor-pointer">
+        <div
+          style={{
+            position: 'relative',
+            border: 'solid',
+            width: '480px',
+            height: '240px',
+            borderRadius: '12px',
+            background: 'rgb(220,220,220)',
+          }}
+        >
+          <HeadlessToast
+            alert={alert}
+            label={label}
+            class={`toast ${start && 'toast-start'} ${top && 'toast-top'} ${
+              center && 'toast-center'
+            } ${end && 'toast-end'} ${middle && 'toast-middle'} ${
+              bottom && 'toast-bottom'
+            }`}
```

Comment:
Something like this `const { sizes, ... } = { sizes: { sm: 'btn-sm', md: 'btn-md', ... } };` is more readable
Full example [here](https://github.com/qwikifiers/qwik-ui/blob/main/CODING_STANDARDS.md#example-of-headless-button-customisation-with-daisy
)

Reply:
This is kind of different because I am choosing several options...
Should I filter the `positions` object to let`s say `chosenPositions` and then flat it to astring to be appended to the class?

Reply:
position could be `'end' | 'middle' | 'bottom'` and then 
{ position: {end: 'toast-end', middle: 'toast-middle' ... } --> position[position]